code/modules/ruins/icemoonruin_code/hotsprings.dm

```diff
@@ -0,0 +1,47 @@
+GLOBAL_LIST_EMPTY(cursed_minds)
```

Comment:
Are minds removing themselves from this list on deletion?

Reply:
no probably not

Reply:
Make sure that happens to avoid issues with garbage collection.

Reply:
no idea what to do with this or what a null reference key in an association list would do

Reply:
The garbage collector wont kill off objects if they’re referenced somewhere, like here. Just remove them from the list once they get qdelled. You could probably add a signal to look for it and remove it when the time comes

Reply:
even if they're hard deleted? strange